apps/api/schema.graphql

```diff
@@ -3,5 +3,6 @@
 # ------------------------------------------------------
 
 type Query {
+  db: String!
```

Comment:
what is the purpose of this?

Reply:
@bassiounix It only exists for making sure that db connection works.